x-pack/plugins/cases/server/saved_object_types/comments.ts

```diff
@@ -127,7 +135,7 @@ export const createCaseCommentSavedObjectType = ({
       },
     },
   },
-  migrations: createCommentsMigrations(migrationDeps),
+  migrations: () => createCommentsMigrations(migrationDeps),
```

Comment:
This is needed so migrations will be created after all plugins' setup lifecycle. This will ensure that when migrations run the registry will contain all registered attachments.